app/controllers/users_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
```

Comment:
You are missing your `create` for user.
How can user signup otherwise?

See screenshot from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49722686/) link
![sign_up_requirement](https://user-images.githubusercontent.com/31193823/69082742-1e791a00-0a17-11ea-948c-45c06fd15212.png)